docs/src/manual/quad_forms/genusherm.md

```diff
@@ -307,23 +415,53 @@ lattices:
 As before, we will construct two different global genus symbols for hermitian
 lattices, which we will use for the rest of this section.
 
-```@repl 2
-using Hecke # hide
-Qx, x = QQ["x"];
-K, a = number_field(x^2 - 2, "a");
-Kt, t  = K["t"];
-E, b = number_field(t^2 - a, "b");
-OK = maximal_order(K);
-p = prime_decomposition(OK, 2)[1][1];
-g1 = genus(HermLat, E, p, [(0, 1, 1, 0), (2, 2, -1, 1)], type = :det);
-infp = infinite_places(E)
-SEK = unique([r.base_field_place for r in infp if isreal(r.base_field_place) && !isreal(r)]);
-length(SEK)
-G1 = genus([g1], [(SEK[1], 1)])
-D = matrix(E, 3, 3, [5//2*a - 4, 0, 0, 0, a, a, 0, a, -4*a + 8]);
-gens = Vector{Hecke.RelSimpleNumFieldElem{AbsSimpleNumFieldElem}}[map(E, [1, 0, 0]), map(E, [a, 0, 0]), map(E, [b, 0, 0]), map(E, [a*b, 0, 0]), map(E, [0, 1, 0]), map(E, [0, a, 0]), map(E, [0, b, 0]), map(E, [0, a*b, 0]), map(E, [0, 0, 1]), map(E, [0, 0, a]), map(E, [0, 0, b]), map(E, [0, 0, a*b])];
-L = hermitian_lattice(E, gens, gram = D);
-G2 = genus(L)
+```jldoctest
+julia> Qx, x = QQ[:x];
+
+julia> K, a = number_field(x^2 - 2, :a);
+
+julia> Kt, t  = K[:t];
+
+julia> E, b = number_field(t^2 - a, :b);
+
+julia> OK = maximal_order(K);
+
+julia> p = prime_decomposition(OK, 2)[1][1];
+
+julia> g1 = genus(HermLat, E, p, [(0, 1, 1, 0), (2, 2, -1, 1)], type = :det);
+
+julia> infp = infinite_places(E)
+3-element Vector{InfPlc{Hecke.RelSimpleNumField{AbsSimpleNumFieldElem}, RelSimpleNumFieldEmbedding{AbsSimpleNumFieldEmbedding, Hecke.RelSimpleNumField{AbsSimpleNumFieldElem}}}}:
+ Infinite place corresponding to (Complex embedding corresponding to root -1.19 of relative number field)
+ Infinite place corresponding to (Complex embedding corresponding to root 1.19 of relative number field)
+ Infinite place corresponding to (Complex embedding corresponding to root 0.00 + 1.19 * i of relative number field)
+
+julia> SEK = unique([r.base_field_place for r in infp if isreal(r.base_field_place) && !isreal(r)]);
+ERROR: type InfPlc has no field base_field_place
```

Comment:
This is broken, but I don't know how this was intended to be

Reply:
I'll push a fix to your branch later today, ok?

Reply:
Works for me, thanks. I won't force-merge anymore